src/chopper_pack.cpp

```diff
@@ -27,6 +27,12 @@ int set_up_and_parse_subparser_split(seqan3::argument_parser & parser, pack_conf
     parser.add_option(config.bins, 'b', "technical-bins",
                       "Into how many technical bins do you want your sequence data to be packed?");
 
+    parser.add_option(config.num_hash_functions, 's', "num-hash-functions",
```

Comment:
I tried this as well, but we can't use -h as the identifier, because it is reserved for the help menu --help. It is actually hard to find a good letter here.

Reply:
Ah well, then we just stick with the old ones.